Public/Sdk/SelfHost/Libraries/MSBuild/msbuild.dsc

```diff
@@ -23,7 +23,6 @@ export const msbuildRuntimeContent = [
     importFrom("SystemRuntimeCompilerServicesUnsafeForMSBuild").withQualifier({targetFramework: "netstandard2.0"}).pkg,
     
     ...BuildXLSdk.isDotNetCoreBuild ? [
-        importFrom("Microsoft.NETCore.App.210").pkg,

```

Comment:
Maybe just check with @smera about this.

Reply:
sorry, missed the intention, why are we removing this one?


---
In reply to: [333269249](https://github.com/microsoft/BuildXL/pull/1024#discussion_r333269249) [](ancestors = 333269249)

Reply:
@smera It's unused since we upgraded to 3.0 and component governance also complains about vulnerabilities due to v2.1:
https://dev.azure.com/mseng/Domino/_componentGovernance/BuildXL.Internal?_a=components&typeId=18775&alerts-view-option=active

Reply:
in that case I would expect that instead of removing that line, we'd have bumped the version to 3.0 for this ref.

Did you actually get all tests passing? I'm surprised the netcore builds didn't fail when removing this

---
In reply to: [333277992](https://github.com/microsoft/BuildXL/pull/1024#discussion_r333277992) [](ancestors = 333277992)

Reply:
Yes bxl and bxl /q:DebugNet472 worked locally

Reply:
"Primary Build" validation should run NetCore build + tests, right?  And it also passed.

Reply:
oh, but net472 is not a dotnetcore build, but a full framework one

---
In reply to: [333279316](https://github.com/microsoft/BuildXL/pull/1024#discussion_r333279316) [](ancestors = 333279316)

Reply:
bxl /q:Debug worked too

Reply:
To be honest, I don't remember exactly the reason for adding all these references. Just that when I did this it was about carefully handcrafting runtime dependencies to have all related tests passing.
So, if you get the rolling build to pass without this reference, I'm good. This runtime content is all about tests, and not related to product bits.

---
In reply to: [333280109](https://github.com/microsoft/BuildXL/pull/1024#discussion_r333280109) [](ancestors = 333280109)

Reply:
All the PR validations passed. 

I'll kick off a rolling build against this branch before we merge it.  I'll update the thread with the result.

Reply:
Here's the [rolling build](https://mseng.visualstudio.com/Domino/_build/results?buildId=10623323&view=results) for this branch

Reply:
All the relevant steps in the rolling build passed, so let's merge this PR

Reply:
great!